src/amo/css/OverallRating.scss

```diff
@@ -0,0 +1,40 @@
+.OverallRating {
+
+  form {
+    display: flex;
+    justify-content: center;
+
+    input {
+      appearance: none;
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    label {
```

Comment:
I'm not too concerned about the performance I mostly think that using a single class name for selectors is just nicer. It makes specificity much simpler and doesn't have that performance issue.

If we always use a single class name then you don't need to think about it. To figure out the selector for the `&.huh` on line 34 you need to read all the way back to line 1 to figure it out. It appears to be `.OverallRating form label.huh`. I just think that writing `.OverallRating-label-love-it` would be nicer and has other advantages.

If you really want to nest the styles to type less you could but being able to look at the class name and know what it does is pretty nice.